spec/features/sign_up_spec.rb

```diff
@@ -12,8 +12,8 @@
     page.fill_in 'user_password_confirmation', with: '123456'
     page.select 'Male', from: 'user_gender'
     click_button  "Sign up"
-    page.accept_prompt( 'The sign up was successful!' )
-    expect(page).to have_current_path(root_path)
+    expect(page).to have_content( 'Only one more step to start enjoying' )
```

Comment:
Consider to start to using I18n ( http://guides.rubyonrails.org/i18n.html ). Is very useful for Internationalization (not your case I guess) and reuse.

You could define your app messages in a single file and use it on different places. If you have to change a message in the future, you only have to replace it once.